lib/task/import/csvImportTask.class.php

```diff
@@ -219,6 +219,9 @@ public function execute($arguments = array(), $options = array())
       // Where to log errors to
       'errorLog' => $options['error-log'],
 
+      // Is the 'update' parameter set
+      'doUpdate' => ($options['update']) ? true : false,
```

Comment:
We also have a variable (declared line 1103 looks like) called 'updateExisting' which seems to be doing the same thing as 'doUpdate' is here. I think you should remove either that or this 'doUpdate' one (I'm noticing updateExisting has a  similar undeclared variable issue as I commented on above--heh). I don't really have a preference which one we should keep.